Basic/Core/Core.pm

```diff
@@ -1204,7 +1206,7 @@ sub PDL::new {
          # new was passed a string argument that doesn't look like a number
          # so we can process as a Matlab-style data entry format.
 		return PDL::Core::new_pdl_from_string($new,$value,$this,$type);
-      } elsif ($Config{ivsize} < 8 && $pack[$new->get_datatype] eq 'q*') {
+      } elsif (! $CAN_PACK_QUAD && $pack[$new->get_datatype] =~ /^[qQ]\*$/ ) {
```

Comment:
This could be `/q/i`?

Reply:
I think you're right, let's also simplify `PACK_D` to use `eval` too.

Reply:
Done and done!